pkg/controllers/cstorvolumeconfig/pdb.go

```diff
@@ -76,6 +77,28 @@ func (c *CVCController) createPDB(poolNames []string, cspcName string) (*policy.
 		Create(context.TODO(), pdbObj, metav1.CreateOptions{})
 }
 
+// createV1PDB creates PDB for cStorVolumes based on arguments
+func (c *CVCController) createV1PDB(poolNames []string, cspcName string) (*policyv1.PodDisruptionBudget, error) {
+	// Calculate minAvailable value from cStorVolume replica count
+	//minAvailable := (cvObj.Spec.ReplicationFactor >> 1) + 1
```

Comment:
Why is this line commented out & doesn't look like a comment but a line of unused variable declaration?

Reply:
It was there to begin with on the v1beta1 function. So I've carried it forward, as any change there should also  be reflected on the v1 function.